### PR TITLE
validate step

### PR DESCRIPTION
**Pull request by aarjay00**
https://simpledatalabs.slack.com/archives/C03BD0C32SC/p1676696401407209

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ To run all unit tests in your Prophecy project containing Python/Scala projects
 pbt test --path /path/to/your/prophecy_project/
 ```
 
+To quickly validate if all your pipelines are not broken
```
Comment (kirankaranth) [inline README.md]: Update setup.py version

File: README.md [hunk 2]
```diff
@@ -86,6 +86,49 @@ Found 1 pipelines: customers_orders (python)
 
 ✅ Unit test for pipeline: pipelines/customers_orders succeeded.
 ```
+To quickly validate if all your pipelines are not broken
```
Comment (kirankaranth) [inline README.md:89]: Add doc to prophecy-docs as well

File: README.md (hunk 1 above)
Comment (aarjay00) [reply README.md]: updated to `1.0.4.0`

File: README.md (hunk 2 above)
Comment (aarjay00) [reply README.md:89]: https://github.com/SimpleDataLabsInc/prophecy-docs/pull/102

**Review by kirankaranth — APPROVED** [review]

**Pull request merged**